crates/sui-graphql-client/src/lib.rs

```diff
@@ -145,6 +184,25 @@ impl Default for PaginationFilter<'_> {
     }
 }
 
+impl<T: Serialize> From<T> for Name {
+    fn from(value: T) -> Self {
+        Name(bcs::to_bytes(&value).unwrap())
+    }
+}
```

Comment:
I don't think this is necessarily correct given a Name includes the type and this would probably only include the value?

Reply:
or you should rename this type to indicate that its the name value and not the full name

Reply:
Ah yes, good catch. Thank you